akka-typed/src/main/scala/akka/typed/internal/ActorCell.scala

```diff
@@ -326,6 +315,23 @@ private[typed] class ActorCell[T](
         else unsafe.getAndAddInt(this, statusOffset, -1)
       }
     }
+
+    try {
+      unscheduleReceiveTimeout()
+      if (!isTerminated(status))
+        process()
+      scheduleReceiveTimeout()
+      afterProcess()
+    } catch {
+      case NonFatal(ex) ⇒
+        fail(ex)
+        afterProcess()
```

Comment:
The question is which kind of failures we are protecting against here. The only thing I can see that could reasonably throw (apart from VM errors) is the executor. Since these are in tail position in any case, why not just ignore them? What else can we reasonably do? Failing the actor is not correct since it did not fail.